tests/core/pyspec/eth2spec/test/altair/block_processing/test_process_sync_committee.py

```diff
@@ -160,13 +161,13 @@ def validate_sync_committee_rewards(spec, pre_state, post_state, committee, comm
                 committee_bits,
             )
 
-            if proposer_index == index:
-                reward += compute_sync_committee_proposer_reward(
-                    spec,
-                    pre_state,
-                    committee,
-                    committee_bits,
-                )
+        if proposer_index == index:
+            reward += compute_sync_committee_proposer_reward(
+                spec,
+                pre_state,
+                committee,
+                committee_bits,
+            )
```

Comment:
coincidentally, it seems the proposer was in the sync committee for all of the other tests using this code...

if we want to check every validator (or at least the case where the proposer is outside the sync committee) then we need to unindent this conditional one level